README.md

```diff
@@ -36,15 +36,15 @@
 
 ## Usage
 
-1. Download the flavor of your choice from [`themes/`](./themes/).
-2. Open the Simplex's settings.
+1. Download the flavor of your choice from [`/themes`](./themes/).
```

Comment:
```suggestion
1. Download the flavor of your choice from [`themes/`](./themes/).
```
The point of the trailing slash is to indicate that it is a directory. It should stay that way, starting slash doesn't seem to indicate anything of value.